<?xml version="1.0" ?>
<launch>
  <param name="use_sim_time" value="false"/>
  <param name="num_robots" type="int" value="3"/>
  <param name="num_targets" type="int" value="2"/>
  <include file="$(find clt_random_datasets)/launch/world.launch"/>
  <group ns="targets">
    <node name="target1" output="screen" pkg="clt_random_datasets" type="ball">
      <rosparam param="initial_pose">[5.201512923788788, -3.0757516368808857, 1.1374927010326905]</rosparam>
      <param name="freq_pub" value="100"/>
    </node>
    <node name="target2" output="screen" pkg="clt_random_datasets" type="ball">
      <rosparam param="initial_pose">[5.352821131816061, 4.380775932524045, 1.407055347169877]</rosparam>
      <param name="freq_pub" value="100"/>
    </node>
  </group>
  <group ns="robots">
    <node name="robot1" output="screen" pkg="clt_random_datasets" type="robot">
      <param name="radius" value="0.5"/>
      <param name="height" value="0.81"/>
      <param name="landmark_range" value="3.5"/>
      <param name="target_range" value="3.5"/>
      <rosparam param="initial_pose">[5.418755308669976, 0.862795708539192, -2.4559709093022084]</rosparam>
      <rosparam param="alphas">[0.015, 0.1, 0.5, 0.001]</rosparam>
    </node>
    <node name="robot2" output="screen" pkg="clt_random_datasets" type="robot">
      <param name="radius" value="0.5"/>
      <param name="height" value="0.81"/>
      <param name="landmark_range" value="3.5"/>
      <param name="target_range" value="3.5"/>
      <rosparam param="initial_pose">[6.713762799370575, -1.3907420279220837, -2.655163923295181]</rosparam>
      <rosparam param="alphas">[0.015, 0.1, 0.5, 0.001]</rosparam>
    </node>
    <node name="robot3" output="screen" pkg="clt_random_datasets" type="robot">
      <param name="radius" value="0.5"/>
      <param name="height" value="0.81"/>
      <param name="landmark_range" value="3.5"/>
      <param name="target_range" value="3.5"/>
      <rosparam param="initial_pose">[1.4355441697220264, 1.699143246916969, 2.407210808207175]</rosparam>
      <rosparam param="alphas">[0.015, 0.1, 0.5, 0.001]</rosparam>
    </node>
  </group>
</launch>
